input-output-hk/lace
# [LW-9186]Fix: take in account multidelegation transactions with inputs that share same txID

# Checklist

- [x] JIRA - LW-9186

---

## Proposed solution
On `gettingTxInputsValueAndAddress` we were only working under the idea that a transaction could have inputs with different txID's, whereas in multidelegation transactions the txID's are all equal, and the addresses are not. Therefore we  take this consideration into account and resolve the values and addresses for the inputs to match the same data in CExplorer.

## Testing
- Open any transaction and check that the inputs match to CExplorer
- Create a delegation transaction and check the inputs match 
- Update your delegation portfolio and check the inputs again 

## Screenshots
<img width="677" alt="Screenshot 2023-11-29 at 16 39 52" src="https://github.com/input-output-hk/lace/assets/4052063/1c1cdbec-3e2c-4fc8-8f7d-386eaa062a42">
<img width="639" alt="Screenshot 2023-11-29 at 16 39 33" src="https://github.com/input-output-hk/lace/assets/4052063/0b84cc92-5e1d-465d-b12f-54c9b0336482">
<img width="630" alt="Screenshot 2023-11-29 at 16 39 24" src="https://github.com/input-output-hk/lace/assets/4052063/317dcda7-949f-40a8-80d2-6edd8cb05dc7">
